test/terraform/permanent/providers.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_version = "1.9.8"
+  required_providers {
+    aws = {
+      version = "5.76.0"
+    }
```

Comment:
Pin versions to minimize discrepancies. Originally I also had lock files in the PR but dealing with hashes for different platforms seems a bit too tedious for our simple use case.